python/cudf_polars/cudf_polars/callback.py

```diff
@@ -145,6 +145,12 @@ def _callback(
         set_device(device),
         set_memory_resource(memory_resource),
     ):
+        if os.environ.get("CUDF_POLARS_DASK", "OFF").upper() == "ON":
+            # Use experimental Dask executor
+            from cudf_polars.experimental.parallel import evaluate_dask
+
+            return evaluate_dask(ir).to_polars()
```

Comment:
*note*: You can put arbitrary keyword arguments into the `pl.GPUEngine` object and handle them here

Reply:
Done in https://github.com/rapidsai/cudf/pull/17262/commits/29b2d7b174e3d1b6c361b78d50c8ddc9142c58b3